rust/Cargo.lock

```diff
@@ -5897,7 +5897,7 @@ checksum = "a2eb9349b6444b326872e140eb1cf5e7c522154d69e7a0ffb0fb81c06b37543f"
 [[package]]
 name = "str0m"
 version = "0.5.1"
-source = "git+https://github.com/firezone/str0m?branch=main#07c4fb0449db1fdde4b3421a61b666312ec81106"
+source = "git+https://github.com/firezone/str0m?branch=main#b59f9eb58a5a5b506e12163b7fb8ccd088f057f1"
```

Comment:
Diff is only an upstream patch that I had to backport to our fork: https://github.com/firezone/str0m/compare/firezone:str0m:07c4fb0449db1fdde4b3421a61b666312ec81106...b59f9eb58a5a5b506e12163b7fb8ccd088f057f1